### ISSUE TITLE
Invalid passport on recently issued valid passport

### ISSUE THREAD

**Issue by akeilox** [jllarraz/AndroidPassportReader#19]
![50FFBCA6-4BF0-4145-8679-6C28E4A0F341](https://user-images.githubusercontent.com/4035501/84313228-cfecc200-ab98-11ea-9591-eaf10513e842.jpeg)

When a January 2020 issued valid passport scanned it shows Invalid passport.
Running the latest version and it scans other passport issued before just fine.

What It could be missing ?

**Comment by akeilox** [comment]
I am using the latest version with the NFC timeout increase already in place.
And I am not doing the CSCA verification either. 

I can get the picture and details of the passport just fine. But it shows Invalid passport strangely when its just a brand new (january 2020 issue) valid passport.

To clarify I dont get Invalid for other passports of same country from previous issues or others.

What could be the issue in this case ?

**Comment by akeilox** [comment]
@jllarraz is there something else I can check for this Invalid passport message on 2020 issued passports?
Like I mentioned above, picture and details read fine but the Invalid/Red marking shows up.

**Comment by RomainL972** [comment]
Hello @akeilox,
By any chance of your passport from France or maybe another European country? Because I have the same problem with my passport issued in December 2019

**Comment by akeilox** [comment]
Hi @RomainL972 its not an European passport, but Asian ones i have tested. 
All of the ones I have tested with this Chip Invalid sign showing are the December 2019 to 2020 **newly issued** passports. It is consistent that when I check another newly issued passport (march 2020) it too gives the same Chip invalid but it reads the picture and details fine.
Does it read the details and picture fine for your case ?

If yes then its a common global issue, and not a european or asian thing.

**Comment by jllarraz** [comment]
The picture is stored in another datagram, so is the basic information. Interesting if they have changed the specification, would like to know which is the new one

**Comment by RomainL972** [comment]
Yes I'm able to read the details and picture, and the chip authentication only works if I explicitly set the encryption algorithm, which I shouldn't have to do as it may not be compatible with all passports

**Comment by akeilox** [comment]
I think we may have something in common with the France and Singapore, in the sense both countries biometric passports are done by the same French company Thales Group www.thalesgroup.com

But looking at their website, they have also done many other country passports too in Europe or Asia, and must have complied with ICAO

**Comment by akeilox** [comment]
This page kinda gives a trail of which countries they have done recently;

https://www.thalesgroup.com/en/markets/digital-identity-and-security/government/customer-cases

What do you think might have changed / updated ?

**Comment by akeilox** [comment]
Some of the articles show company name Gemalto in the link above, which was acquired by Thales Group. US, UK etc. most prominent ones seems to be made by them.
